bin/check-elb-sum-requests.rb

```diff
@@ -132,7 +121,6 @@ def check_sum_requests(elb)
 
     @severities.each_key do |severity|
       threshold = config[:"#{severity}_over"]
-      puts metric_value
```

Comment:
why is this being removed?

Reply:
I removed this because it looks like a debug statement that was left and forgot. See example output:
```
$ ./check-elb-sum-requests.rb -a <key> -k <secret> -r eu-west-1
264.0
264.0
CheckELBSumRequests OK: <AWS::ELB::LoadBalancer name:my_alb>; ( within 60 seconds between 2018-10-28 07:55:37 +0000 to 2018-10-28 07:56:37 +0000)
```